README.md

```diff
@@ -57,6 +57,21 @@ A diagram of this running 5 steps on a single sample image may help explain how
 ## Installation
 
 This library conveniently works in the browser, in Node, and on the command line (CLI).
+To install the app, run the following commands in a command line:
+
+* `git clone https://github.com/vaibhavmatta/image-sequencer`
+* `cd image-sequencer`
```

Comment:
also instead of just this maybe you can add " then navigate into the cloned folder" and then write the command

Reply:
Alright. I will make this change. Thanks for the suggestions.

Reply:
cd works on every OS. This shouldn't matter.

Reply:
yeah that I know, suggesting to add a line before the command , but this way works well

Reply:
oh ok. I thought you asked to remove the code.